syntax = "proto2";
enum BirdType {
    Chicken = 1;
    Duck = 2;
    Goose = 3;
}
